scipy/stats/morestats.py

```diff
@@ -2713,7 +2714,7 @@ def mood(x, y, axis=0):
 
 
 def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
-             alternative="two-sided"):
+             alternative="two-sided", method='auto'):
```

Comment:
I noticed that `kstest` and `ks_2samp` have a similar parameter, but it is called `mode`.  While I like `method` better, we should probably use `mode` here (and in other functions in the future when we make similar enhancements) to follow the precedent of the KS functions and maintain a consistent API.  @pvanmulbregt, @rlucas7, @mdhaber, others: any opinions on this?